tools/clang/test/DXC/include-main.test

```diff
@@ -0,0 +1,13 @@
+// Test display include process with /Vi
```

Comment:
This file looks like an `hlsl` file, not a test file

Reply:
I'm not sure I agree with how you changed this. Instead of changing the extension to `hlsl` you moved the code out to a separate file that is only used from one test. That seems unnecessary.

Reply:
This file also tests hlsl file not in current directory which includes header file in different directory.
So I want to keep it as a test instead of hlsl.

Reply:
I'm not sure I fully understand the value of that test (since I would hope we have alternative tests covering that case), but it could also be accomplished in a more clearly deliberate way by adding `cd %T && ` in front of the dxc command.

Reply:
Updated with cd %T.